.travis.yml

```diff
@@ -17,7 +13,6 @@ install:
   - nvm --version
   - node --version
   - npm --version
-  - 'if [ "${TRAVIS_NODE_VERSION}" = "0.8" ] ; then nvm install --latest-npm 0.8 ; fi'
   - npm install
   - node --version
```

Comment:
Extra `node --version` & `npm --version` steps can go away if we drop the conditional `nvm install --latest-npm 0.8` step.